src/laundryScraper.py

```diff
@@ -0,0 +1,60 @@
+"""Libraries for reading in webpages and parsing them"""
+import urllib3
+from bs4 import BeautifulSoup
+
+"""Include building and machine classes"""
+from building import *
+from machine import *
+
+"""Read in the LaundryView page for Barton Hall then use BeautifulSoup to convert it into a parsable object"""
+http = urllib3.PoolManager()
+barton = http.request('GET', 'http://classic.laundryview.com/classic_laundry_room_ajax.php?lr=3695891&adk=155604%27')
```

Comment:
Pull out the URL into a couple of variables, one which is the formatting for the URL and one which is the laundry room code. That will make it easier later when we run it on multiple rooms.

Like this:

```
url = 'http://classic.laundryview.com/classic_laundry_room_ajax.php?lr={}'
laundryRoom = '3695891'
building = http.request('GET', url.format(laundryRoom))
```

Side note: Do we have any idea what the `&adk=155604%27` does? I tried messing with it a bit and it didn't seem to do anything. Maybe we can leave it out.